frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/executions.cy.ts

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable camelcase */
+import {
+  buildMockPipelineV2,
+  buildMockPipelines,
+  mockDashboardConfig,
+  mockDataSciencePipelineApplicationK8sResource,
+  mockK8sResourceList,
+  mockProjectK8sResource,
+  mockRouteK8sResource,
+} from '~/__mocks__';
+import {
+  DataSciencePipelineApplicationModel,
+  RouteModel,
+  ProjectModel,
+} from '~/__tests__/cypress/cypress/utils/models';
+import {
+  executionPage,
+  executionFilter,
+} from '~/__tests__/cypress/cypress/pages/pipelines/executions';
+import { mockGetExecutions, mockGetNextPageExecutions } from '~/__mocks__/mlmd/mockGetExecutions';
+import { tablePagination } from '~/__tests__/cypress/cypress/pages/components/Pagination';
+import { initMlmdIntercepts } from './mlmdUtils';
+
+const projectName = 'test-project-name';
+const initialMockPipeline = buildMockPipelineV2({ display_name: 'Test pipeline' });
+
+describe('ExecutionsError', () => {
+  it('Fails to load executions list', () => {
+    initIntercepts(false);
+    executionPage.visit(projectName);
+    executionPage.findText('There was an issue loading executions');
+  });
+});
+
+describe('No Executions', () => {
+  it('Has no executions', () => {
+    initIntercepts(true, true);
+    executionPage.visit(projectName);
+    executionPage.findText('No executions');
+  });
+});
+
+describe('Executions', () => {
+  beforeEach(() => {
+    initIntercepts(true);
+    executionPage.visit(projectName);
+  });
+
+  it('Makes filter request', () => {
+    shouldFilterItems(FilterArgs.Execution, 'h');
+    shouldFilterItems(FilterArgs.ID, '2');
+    shouldFilterItems(FilterArgs.Status);
+    shouldFilterItems(FilterArgs.Type);
+  });
+
+  it('Includes more entries and navigates pages', () => {
+    setUpIntercept();
+    tablePagination.top.selectToggleOption('20 per page');
+    cy.wait('@request');
```

Comment:
You should assert that the correct payload or query params were sent along with the request and not just that a request was made.

Reply:
Ah yes, I'm not sure if this is possible to do with gRPC requests. I'm going to ask around if we have a current solution for this, as I can't seem to find one in past PRs.

Reply:
If there's no current solutions, we can also create a new ticket to do that work with the stipulation that we circle back and update these tests cases.

Reply:
I figured out a way! PR should be ready in an hour or so

Reply:
I only was able to fix the filter ones